workspaces/libnpmteam/lib/index.js

```diff
@@ -47,12 +47,12 @@ cmd.rm = (user, entity, opts = {}) => {
   const { scope, team } = splitEntity(entity)
   validate('SSO', [scope, team, opts])
   const uri = `/-/team/${eu(scope)}/${eu(team)}/user`
-  return npmFetch.json(uri, {
+  return npmFetch(uri, {
     ...opts,
     method: 'DELETE',
     scope,
     body: { user },
-  })
+  }).then(res => res.text())
```

Comment:
npm doesn't parse the results of delete operations, there is no need to call `.text` here or above.

See https://github.com/npm/cli/blob/latest/workspaces/libnpmaccess/lib/index.js#L121-L129 for how we do this in other modules